tasks/mina/git.rb

```diff
@@ -48,6 +50,16 @@
     run :local do
       comment %{Ensuring everything is pushed to git}
       command %{
+        if [ $(git branch --list #{fetch(:branch)} | wc -l) -eq 0 ]; then
+          echo "! #{fetch(:git_local_branch_missing_message)}"
+          exit 1
+        fi
```

Comment:
is this check needed?
because i can do `mina deploy branch=some_branch` without actually fetching it from git.

I have a feeling remote_branch check is enough

Reply:
Ohh, I see. Cool, carry on